catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -285,4 +313,64 @@ public boolean matches(Object o) {
             return test.test((QueryRequest) o);
         }
     }
+
+    private void mockMultipleCatalogFrameworkQueries(int pageSize, int resourceLimit)
+            throws Exception {
+
+        mockResourceResponse();
+        QueryResponse queryResponseMock = mock(QueryResponse.class);
+
+        // Populate list of mock results, sized at resource limit
+        List<Result> resultsList = populateResultMockList(resourceLimit);
+        List<Result>[] queriedResultsList = mockQueryGetResults(resultsList, pageSize);
+
+        when(catalogFramework.query(any())).thenReturn(queryResponseMock);
+        when(queryResponseMock.getResults()).thenReturn(queriedResultsList[0], queriedResultsList);
```

Comment:
✏️ I believe the second value in `thenReturn` should be `queriedResultsList[1]`. Otherwise, the mock behavior would be equivalent to:
```java
when(queryResponseMock.getResults())
    .thenReturn(queriedResultsList[0])
    .thenReturn(queriedResultsList[0])
    .thenReturn(queriedResultsList[1]);
```